src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -258,6 +289,21 @@ def get(self, request: Request, project: Project) -> Response:
                     # When no code mapping have been matched we have not attempted a URL
                     if current_config["outcome"].get("attemptedUrl"):
                         result["attemptedUrl"] = current_config["outcome"]["attemptedUrl"]
+
+                result["commitSha"] = None
+                if features.has(
```

Comment:
We probably want to check `organization.flags.codecovAccess` here as well. I think we'll always need to check both the flag and the setting

Reply:
Thank you, this is a good idea